airbyte-integrations/connectors/source-enquire-labs/source_enquire_labs/spec.json

```diff
@@ -0,0 +1,33 @@
+{
+  "documentationUrl": "https://docsurl.com",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Enquire Labs Spec",
+    "type": "object",
+    "required": ["secret_key"],
+    "additionalProperties": false,
+    "properties": {
+      "secret_key": {
+        "type": "string",
+        "description": "secret key of enquire labs",
+        "airbyte_secret": true
+      },
+      "since": {
+        "type": "string",
+        "description": "Use this ISO8601 formatted timestamp to fetch responses that are newer than the supplied value",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}$",
+        "example": ["2020-07-10 15:00:00.000"]
+      },
+      "until": {
+        "type": "string",
+        "description": "Use this ISO8601 formatted timestamp to fetch responses that are older than the supplied value",
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2} [0-9]{2}:[0-9]{2}:[0-9]{2}$",
+        "example": ["2020-07-10 15:00:00.000"]
+      },
+      "question_id": {
+        "type": "integer",
+        "description": ""
```

Comment:
Could you explain the role of this parameter in the description? Do you think it should really be an integer or a string could work too ?